nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -59,6 +62,9 @@ DockingServer::on_configure(const rclcpp_lifecycle::State & state)
   get_parameter("fixed_frame", fixed_frame_);
   get_parameter("dock_backwards", dock_backwards_);
   get_parameter("dock_prestaging_tolerance", dock_prestaging_tolerance_);
+  get_parameter("initial_rotation", initial_rotation_);
+  get_parameter("initial_rotation_min_angle", initial_rotation_min_angle_);
+  get_parameter("dock_backwards_without_sensor", dock_backwards_without_sensor_);
   RCLCPP_INFO(get_logger(), "Controller frequency set to %.4fHz", controller_frequency_);
```

Comment:
Check if the dock backward without sensor is set but the dock backwards is not. In that case, return a failure on the configuration since that's an invalid configuration that is impossible to disambiguate